CONTRIBUTING.md

```diff
@@ -113,7 +61,7 @@ A short list of tools used in `dbt-core` testing that will be helpful to your un
 
```

Comment:
Should we add `changie` to this list of tools?  And describe them as tools for development, not tools for testing?